charts/gateway-addons-helm/values.yaml

```diff
@@ -53,3 +61,173 @@ prometheus:
     image:
       repository: prom/prometheus
     securityContext: {}
+
+
+# Values for Fluent-bit dependency
+fluent-bit:
+  enabled: true
+  fullnameOverride: fluent-bit
+  testFramework:
+    enabled: false
+  podAnnotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "2020"
+    prometheus.io/path: /api/v1/metrics/prometheus
+    fluentbit.io/exclude: "true"
+  ## https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/configuration-file
+  config:
+    service: |
+      [SERVICE]
+          Daemon Off
+          Flush {{ .Values.flush }}
```

Comment:
these templates are being filled by `fluent-bit` or our tooling ?

Reply:
these values are directly copied from https://github.com/envoyproxy/gateway/blob/main/examples/fluent-bit/helm-values.yaml, same for others.

the values under example dir will be removed within follow-up PR.